tests/ccl_test_cosmology.c

```diff
@@ -41,36 +41,17 @@ CTEST_SETUP(cosmology) {
 // Check to see if general ccl_cosmology struct is initialized correctly
 CTEST2(cosmology, create_general_cosmo) {
   ccl_configuration config = default_config;
-  
-  // Initialize ccl_cosmology struct
-  ccl_cosmology * cosmo = ccl_cosmology_create_with_params(data->Omega_c, data->Omega_b, data->Omega_k, 
-							   data->Neff, &(data->mnuval), data->mnu_type,
-							   data->w0, data->wa, data->h, data->A_s, data->n_s,
-							   -1,-1,-1,-1, NULL, NULL, config, &(data->status));
-  
-  // Pull ccl_parameters object out of ccl_cosmology
-  ccl_parameters params = (*cosmo).params;
-  
-  ASSERT_DBL_NEAR_TOL(params.Omega_c, data->Omega_c, 1e-10);
-  ASSERT_DBL_NEAR_TOL(params.w0, -1.0, 1e-10);
-  ASSERT_DBL_NEAR_TOL(params.wa, data->wa, 1e-10);
-  ASSERT_DBL_NEAR_TOL(params.bcm_etab, 0.5, 1e-10);
-  ASSERT_DBL_NEAR_TOL((params.mnu)[0], data->mnuval, 1e-10);
-}
+  ccl_parameters params = ccl_parameters_create_flat_lcdm(
+    data->Omega_c, data->Omega_b, data->h, data->A_s, data->n_s,
+    &(data->status));
 
-// Check to see if LCDM ccl_cosmology struct is initialized correctly
-CTEST2(cosmology, create_lcdm_cosmo) {
-  ccl_configuration config = default_config;
-  
   // Initialize ccl_cosmology struct
-  ccl_cosmology * cosmo = ccl_cosmology_create_with_lcdm_params(data->Omega_c, data->Omega_b, data->Omega_k,
-								data->h,data->A_s, data->n_s, config,
-								&(data->status));
-  
-  // Pull ccl_parameters object out of ccl_cosmology
-  ccl_parameters params = (*cosmo).params;
-  
-  ASSERT_DBL_NEAR_TOL(params.Omega_c, data->Omega_c, 1e-10);
-  ASSERT_DBL_NEAR_TOL(params.w0, -1.0, 1e-10);
-  ASSERT_DBL_NEAR_TOL(params.wa, 0.0, 1e-10);
+  ccl_cosmology * cosmo = ccl_cosmology_create(params, config);
+
+  // IDK what to test here
```

Comment:
This needs to be cleaned up.

Reply:
I can remove the comment, but otherwise there is not a lot to test. lol

Reply:
Do you have a specific suggestion @philbull ?

Reply:
I removed the comment. LMK if you have other requests!